index.html

```diff
@@ -136,7 +136,7 @@ <h1 id="logo"><a href="index.html">Twenty <span>by HTML5 UP</span></a></h1>
 										<header>

```

Comment:
Argh gross! I didn't know it had changed back to tabs. Have changed it back to spaces now.